src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowController.cs

```diff
@@ -123,14 +123,14 @@ public async Task<IActionResult> Index(int workflowTypeId, WorkflowIndexViewMode
             var pagerShape = (await New.Pager(pager)).TotalItemCount(await query.CountAsync()).RouteData(routeData);
             var pageOfItems = await query.Skip(pager.GetStartIndex()).Take(pager.PageSize).ListAsync();
 
+            var workflowIds = pageOfItems.Select(item => item.WorkflowId);
+            var workflows = await _session.Query<Workflow, WorkflowIndex>(item => item.WorkflowId.IsIn(workflowIds))
+                .ListAsync();
+
             var viewModel = new WorkflowIndexViewModel
             {
                 WorkflowType = workflowType,
-                Workflows = pageOfItems.Select(x => new WorkflowEntry
-                {
-                    Workflow = x,
-                    Id = x.Id
-                }).ToList(),
+                Workflows = workflows.Select(x => new WorkflowEntry { Workflow = x, Id = x.Id }).ToList(),
```

Comment:
I like the how it was (for clarify), I use single line for one property assignment